src/include/liburing.h

```diff
@@ -18,6 +18,8 @@
 #include "liburing/io_uring.h"
 #include "liburing/io_uring_version.h"
 #include "liburing/barrier.h"
+#include "../sanitize.h"
+
```

Comment:
This include breaks `make install`.

```sh
ammarfaizi2@integral2:~/lbq$ sudo make install
ammarfaizi2@integral2:~/lbq$ cat q.c
#include <liburing.h>
ammarfaizi2@integral2:~/lbq$ gcc q.c
In file included from q.c:1:
/usr/include/liburing.h:21:10: fatal error: ../sanitize.h: No such file or directory
   21 | #include "../sanitize.h"
      |          ^~~~~~~~~~~~~~~
compilation terminated.
```

You need to ship the "sanitize.h" to the installation path as well.

Reply:
Nice catch, should be added now.